### PR TITLE
Auto size title text in navigation bars and toolbars and custom color

### PR DESCRIPTION
**Pull request by demerino**
Fix for #19 

Use a `UILabel` that will auto resize to fit the title space.

### CODE REVIEW

**Comment by demerino** [comment]
Added optional `navigationBarTitleTextColor` in `BTUIKAppearance` as requested in https://github.com/braintree/braintree-ios-drop-in/issues/22

File: BraintreeUIKit/Helpers/BTUIKAppearance.m [hunk 1]
```diff
@@ -90,6 +90,17 @@ + (void) styleSystemLabelSecondary:(UILabel *)label {
     label.textColor = [BTUIKAppearance sharedInstance].secondaryTextColor;
 }
 
++ (UILabel *) styledNavigationTitleLabel {
```
Comment (skunkworks) [inline BraintreeUIKit/Helpers/BTUIKAppearance.m]: Whitespace

File: BraintreeUIKit/Helpers/BTUIKAppearance.m [hunk 2]
```diff
@@ -90,6 +90,17 @@ + (void) styleSystemLabelSecondary:(UILabel *)label {
...
+    UILabel* tlabel = [[UILabel alloc] initWithFrame:CGRectMake(0,0, 200, 40)];
```
Comment (skunkworks) [inline BraintreeUIKit/Helpers/BTUIKAppearance.m]: `UILabel *tlabel`

File: UnitTests/BTUIKAppearanceTests.m [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+#import "BTUIKAppearance.h"
+
+@interface BTUIKAppearanceTests : XCTestCase
+
+@end
+
+@implementation BTUIKAppearanceTests
+
+- (void)setUp {
```
Comment (skunkworks) [inline UnitTests/BTUIKAppearanceTests.m]: Remove setUp and tearDown

**Pull request merged**